server/rooms.ts

```diff
@@ -485,7 +485,7 @@ export abstract class BasicRoom {
 		}
 		if (!(userGroup in Config.groups)) return false;
 		if (!(modjoinGroup in Config.groups)) throw new Error(`Invalid modjoin setting in ${this.roomid}: ${modjoinGroup}`);
-		return Config.groups[userGroup].rank >= Config.groups[modjoinGroup].rank;
+		return Config.groups[userGroup].rank >= Config.groups[modjoinGroup].rank || user.isSysop;
```

Comment:
I'd add `if (user.isSysop) return true` to the top - sysop checks up bail early, they dont really get checkout alone with other things

Reply:
> they dont really get checkout alone with other things

?

Bailing out early is good though, I'll change that.

Reply:
LOL this not entirely sure where I was going with that sentence either

Reply:
"they don't really need to get checked along with the other things"?